config/config.go

```diff
@@ -68,5 +70,22 @@ func New() (*Config, error) {
 
 	c.ClusterName = os.Getenv("CLUSTER_NAME")
 
-	return c, nil
+	whitelist, err := ParseIPWhitelist(os.Getenv("IP_WHITELIST"))
```

Comment:
This is dangerous error handling IMHO, as if someone adds a new operation with a returned error after this it's easy to forget to handle this error.